manifests/0000_90_kube-controller-manager-operator_03_servicemonitor.yaml

```diff
@@ -42,3 +42,21 @@ spec:
       for: 15m
       labels:
         severity: critical
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: kube-controller-manager-operator
+  namespace: openshift-kube-controller-manager-operator
+spec:
+  groups:
+  - name: cluster-version
+    rules:
+    - alert: KubeControllerManagerDown
+      annotations:
+        message: KubeControllerManager has disappeared from Prometheus target discovery.
+      expr: |
+        kube_poddisruptionbudget_status_expected_pods <= on(poddisruptionbudget, service) kube_poddisruptionbudget_status_desired_healthy
+      for: 15m
+      labels:
+        severity: warning
```

Comment:
I spoke with monitoring team, we don't have info level, so I went with a warning. I'm open to suggestions, though.

Reply:
Wait.... I know there are info level things.  I swear I created one...

Reply:
Where, I've been looking for them and monitoring team explicitly told me there none.

Reply:
Hrm, I guess I was wrong.  This is fine, and is definitely an alert.